WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Deposits Overview/WooPaymentsDepositsCurrencyOverviewViewModel.swift

```diff
@@ -41,7 +53,11 @@ final class WooPaymentsDepositsCurrencyOverviewViewModel: ObservableObject {
     @Published var tabTitle: String
 
     private func formatAmount(_ amount: NSDecimalNumber) -> String {
-        return numberFormatter.string(from: amount) ?? ""
+        if let wooCurrencyFormatter {
+            return wooCurrencyFormatter.formatAmount(amount, numberOfDecimals: 2) ?? ""
```

Comment:
Should we unwrap and use `currencySettings.fractionDigits` in `numberOfDecimals` if this exists, rather than setting them to `2`? I'm thinking of currencies that do not have decimals, like JPY.

Reply:
Great catch, thank you!

Reply:
@iamgabrielma Also we can just leave it blank; the currency formatter will use the number of decimals from the currency settings if nil (default) is passed.